thrift/atoms/tenfour.thrift

```diff
@@ -0,0 +1,61 @@
+namespace * tenfour
+
+include "../shared.thrift"
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+struct Content {
+  // do we need to store the ID, seeing as it is replicated(?) in the
+  // content-atom wrapping?
+  1  : required string id
+  2  : required string title
+  3  : required string updatedBy
+  4  : required DateTime updatedAt
+  5  : required DateTime createdAt
+  6  : required string createdBy
+  7  : required bool published
+  8  : required string quizType
+  9  : optional i16 defaultColumns
+  10 : required ItemContent content
+}
+
+// will vary depending on item's type
+union ItemContent {
+  1: QuizContent quiz
+}
+
+struct QuizContent {
+  1: required list<Question> questions
+  2: required ResultGroups resultGroups
+}
+
+struct ResultGroups {
+  1: required list<ResultGroup> groups
+  2: required bool revealAtEnd
+}
+
+struct ResultGroup {
+  1: required string title
+  2: required string share
+  3: required i16 minScore
+}
+
+struct Question {
+  1: required string questionText
```

Comment:
I forgot to mention it yesterday, and it isn't necessarily related to this (besides the naming), but I was surprised you said this would be plain text rather than HTML. What would be the strategy to support HTML in the future (I struggle to believe it will never become a requirement :-) )?